spirv/autogen_spirv.rs

```diff
@@ -17,112 +17,112 @@
 // DO NOT MODIFY!
 
 pub type Word = u32;
-pub const MAGIC_NUMBER: u32 = 0x07230203;
+pub const MAGIC_NUMBER: u32 = 119734787;
```

Comment:
we shouldn't change the hex output in this module, a regression?

Reply:
That sent me down a rabbit hole but it's back to hex now. `proc_macro2` really doesn't like hex literals.